Subject: Search for time.h, and use it in !HAVE_WAIT3 case.
 This helps getting a correct define for HZ when wait3 is detected to not be
 properly filling in rusage.
Author: Pino Toscano

--- a/configure.in
+++ b/configure.in
@@ -11,7 +11,7 @@ AC_PROG_INSTALL
 dnl Checks for header files.
 AC_HEADER_STDC
 AC_HEADER_SYS_WAIT
-AC_CHECK_HEADERS(unistd.h string.h sys/rusage.h sys/resource.h)
+AC_CHECK_HEADERS(unistd.h string.h sys/rusage.h sys/resource.h time.h)
 
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
--- a/resuse.c
+++ b/resuse.c
@@ -25,6 +25,9 @@
 
 #if !HAVE_WAIT3
 # include <sys/times.h>
+# if HAVE_TIME_H
+#  include <time.h>
+# endif
 # ifndef HZ
 #  include <sys/param.h>
 # endif
